### DISCUSSION TITLE
"Send feedback" should allow to immediately give feedback

### DISCUSSION THREAD

**Discussion by ldcMasa** [community/community#61729] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

My first reaction after clicking "Send feedback" was something like:
Where am I? What is this page?
I expect a feedback page to show a form where you can give feedback, not something like what is currently being shown, which makes no sense to me.
So either don't call this "Send feebdack" or make it a real send feedback page.
It's also not clear that the Send feedback button is only fo the "For you" (default page shown, which I am utterly uninterested in) and not also available for "Following" or as a general feedback button.

**Comment by moonfury-ops** [comment]
Thank you
